SPECS/protobuf/protobuf.spec

```diff
@@ -1,17 +1,26 @@
+%global protoc_path %{_builddir}/%{name}-%{version}/cmake/protoc
+
 Summary:        Google's data interchange format
 Name:           protobuf
-Version:        3.17.3
-Release:        2%{?dist}
+Version:        25.3
```

Comment:
Note: versions `25.3`, `3.25.3`, and `4.25.3` are the same source commit, so this can be read as a jump from `3.17.3` to `3.25.3`.